app/src/atoms/buttons/ODD/MediumButtonRounded.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components'
+import {
+  TYPOGRAPHY,
+  COLORS,
+  SPACING,
+  BORDERS,
+  NewPrimaryBtn,
+  styleProps,
+} from '@opentrons/components'
+
+export const MediumButtonRounded = styled(NewPrimaryBtn)`
+  background-color: ${COLORS.blueEnabled};
+  border-radius: ${BORDERS.size_six};
+  box-shadow: none;
+  height: '4.25rem';
+  font-size: ${TYPOGRAPHY.fontSize28};
+  font-weight: ${TYPOGRAPHY.fontWeightSemiBold};
+  line-height: ${TYPOGRAPHY.lineHeight36};
+  padding-bottom: ${SPACING.spacing4};
+  padding-left: ${SPACING.spacing6};
+  padding-top: ${SPACING.spacing4};
+  padding-right: ${SPACING.spacing6};
```

Comment:
I think we can use a shorthand for these lines.
https://developer.mozilla.org/en-US/docs/Web/CSS/padding
```
/* top and bottom | left and right */
padding: 5% 10%;
```

```suggestion
  padding: ${SPACING.spacing4} ${SPACING.spacing6};
```